File: com/problem/go/googleKickstart/2022/roundg/FindCandy.go
package main

import (
  "fmt"
)

func solve(N int,M int) {
  /**
  * 1. sum all of the candies
  * 2. divide total candies to M kids
  */
  var total int
  for i := 0; i < N; i++ {
    var S int
    fmt.Scanf("%d", &S)
    total += S;
  }
  
  fmt.Println(total % M)
}

func main() {
  var T int
  fmt.Scanf("%d", &T)
  for t := 1; t <= T; t++ {
    var N int
    var M int
    fmt.Scanf("%d %d", &N, &M)
    fmt.Printf("Case #%d: ", t)
    solve(N, M)
  }
}

File: com/problem/cpp/googleKickStart/session/3/FindCandy.cpp
 #include <iostream>
 using namespace std;
 
 int solve() {
    int n,m = (cin >> n >> m,n,m);
    int sum = 0;
    while(n--) {
        int c = (cin >> c,c);
        sum += c;
    }
    return sum % m;
 }
 
 int main() {
    int t = (cin >> t,t);
    for (int tt=1; tt<=t; tt++) 
        printf("Case #%d: %d\n",tt,solve());
    return 0;
 }
 
 